### DISCUSSION TITLE
How to process and image with ONNX?

### DISCUSSION THREAD

**Discussion by Riczap** [microsoft/onnxruntime#13713] [Other Q&A]
I built a bot that uses computer vision and YOLOv5 on Python, but I'm trying to remake it on C++. I already have my working 'model.onnx' but I have no clue on how to use the ONNX api.
I'm trying to understand the API on python first, and I get that I need to use the following:
`session = rt.InferenceSession("best.onnx")`
`outputs = session.run(output_names, input_feed)`
I have jpg files that I pass with cv2.imread() but I don't know how to use them on the `run` parameters.

**Comment by faxu** [comment]
There are some samples for C++ here: https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx
Here is a Python example including loading an image: https://github.com/onnx/onnx-docker/blob/master/onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb

**Reply by Riczap** [reply]
Thanks! I'll check them out and see what I can do.
